hack/jenkins/upload-to-gcs.sh

```diff
@@ -151,15 +151,22 @@ function update_job_result_cache() {
 
   mkdir -p ${tmp_results%/*}
 
+  # Construct a valid json file
```

Comment:
Some of the jq queries I was making were barfing and when I added this it started working.  If you're jq-foo is good and you want to take a look, I can send you some details?  But this also works, albeit a bit less simple than before.